components/com_finder/tmpl/search/default_result.php

```diff
@@ -115,7 +116,18 @@
                     <?php endforeach; ?>
                     <?php if (count($taxonomy_text)) : ?>
                         <li class="result__taxonomy-item result__taxonomy--<?php echo $type; ?>">
-                            <span><?php echo Text::_(LanguageHelper::branchSingular($type)); ?>:</span> <?php echo implode(',', $taxonomy_text); ?>
+                            <span><?php echo Text::_(LanguageHelper::branchSingular($type)); ?>:</span>
+                            <?php
+                            $text = implode(',', $taxonomy_text);
+                            ?>
```

Comment:
```suggestion
                            <?php $text = implode(',', $taxonomy_text); ?>
```
No need to have that in 3 lines here, I think.

Reply:
Thanks!